drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -472,6 +473,21 @@
   font-size: 0.875em;
 }
 
+.xblock--drag-and-drop .btn-link {
+    padding: 1px;
+    font-size: 0.875em;
+}
+
+.xblock--drag-and-drop .btn-default.btn-link:hover,
```

Comment:
I understand limiting scope, but this PR is actually introducing this css. I'd rather not see custom handling of focus indicators added-- it's much better to use the default platform styling (which may be different on different browsers).